android/java/org/chromium/chrome/browser/BackgroundVideoPlaybackTabHelper.java

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser;
+
+import org.jni_zero.JNINamespace;
+import org.jni_zero.NativeMethods;
+
+import org.chromium.content_public.browser.WebContents;
+
+@JNINamespace("chrome::android")
+public class BackgroundVideoPlaybackTabHelper {
+
+    public static void sendOrientationChangeEvent(WebContents webContents, boolean isFullScreen) {
+        BackgroundVideoPlaybackTabHelperJni.get()
+                .sendOrientationChangeEvent(webContents, isFullScreen);
+    }
+
+    public static boolean isPlayingMedia(WebContents webContents) {
+        return BackgroundVideoPlaybackTabHelperJni.get().isPlayingMedia(webContents);
+    }
+
+    @NativeMethods
+    interface Natives {
+        void sendOrientationChangeEvent(WebContents webContents, boolean isFullScreen);
+
+        boolean isPlayingMedia(WebContents webContents);
```

Comment:
Do we need to expose this function ? can we check idf the media is being played before we inject the script ?

Reply:
we are not injecting the script in this case. we are checking only when user press the home button. we have to display PIP on portrait mode if video is playing.

Reply:
We have to display PIP from portrait mode and in that case `FullscreenVideoPictureInPictureController` is not handling this.